packages/block-editor/src/components/use-block-commands/index.js

```diff
@@ -242,49 +204,73 @@ const useActionsCommands = () => {
 			callback: onGroup,
 			icon: group,
 		},
-		{
-			name: 'move-to',
-			label: __( 'Move to' ),
-			callback: onMoveTo,
-			icon: move,
-		},
-		{
-			name: 'add-after',
-			label: __( 'Add after' ),
-			callback: onAddAfter,
-			icon: add,
-		},
-		{
-			name: 'add-before',
-			label: __( 'Add before' ),
-			callback: onAddBefore,
-			icon: add,
-		},
-		{
+	];
+	if ( canInsertDefaultBlock ) {
+		commands.push(
+			{
+				name: 'add-after',
+				label: __( 'Add after' ),
+				callback: () => {
+					const clientId = Array.isArray( clientIds )
+						? clientIds[ clientIds.length - 1 ]
+						: clientId;
+					insertAfterBlock( clientId );
+				},
+				icon: add,
+			},
+			{
+				name: 'add-before',
+				label: __( 'Add before' ),
+				callback: () => {
+					const clientId = Array.isArray( clientIds )
+						? clientIds[ 0 ]
+						: clientId;
+					insertBeforeBlock( clientId );
+				},
+				icon: add,
+			}
+		);
+	}
+	if ( canRemove ) {
```

Comment:
I think we also need to check that there is more than one block, just as [the Move to action on the block toolbar does](https://github.com/WordPress/gutenberg/blob/3a7404dd80067715da65caffb7f830b3ca218617/packages/block-editor/src/components/block-settings-menu-controls/index.js#L93).